stories/Templates/LandingPage/GenericLandingPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import './genericlandingpage.scss';
+import { PageHeroOption } from '../../Organism/Blocks/Heros/Internal-Page-Heros/PageHeroOption';
+import  FilterAndSearchBar  from "../../Organism/FilterAndSearchBar/FilterAndSearchBar";
+import { ContentCard } from '../../Organism/Blocks/ContentCard/ContentCard';
+import { Footer } from '../../Organism/Footer/Footer';
+import logo from '../../assets/images/logo.svg';
+import logowhite from '../../assets/images/logo-white.svg';
+
+
+export const GenericLandingPage = ({ data, active, herodata, title, imgsrc, videosrc, footerData, content, groupcontentdata, buttonname}) => (
+  <div>
```

Comment:
@neelesh-tippe div is not required here

Reply:
@neelesh-tippe If you need div here then instead of that we can use the fragment <></>